libsolidity/formal/PredicateInstance.h

```diff
@@ -37,7 +37,7 @@ smtutil::Expression interface(Predicate const& _pred, ContractDefinition const&
 smtutil::Expression nondetInterface(Predicate const& _pred, ContractDefinition const& _contract, EncodingContext& _context, unsigned _preIdx, unsigned _postIdx);
 
 smtutil::Expression constructor(Predicate const& _pred, EncodingContext& _context);
-smtutil::Expression constructorCall(Predicate const& _pred, EncodingContext& _context);
+smtutil::Expression constructorCall(Predicate const& _pred, EncodingContext& _context, bool _internal = true);
```

Comment:
As mentioned in our call: the precise condtions for ``_internal`` being true and false would be nice to have in a comment.